dialogplus/src/main/java/com/orhanobut/dialogplus/DialogPlus.java

```diff
@@ -256,6 +276,16 @@ public void onAnimationEnd(Animation animation) {
                     public void run() {
                         decorView.removeView(rootView);
                         isDismissing = false;
+                        if (onDismissListener == null) {
+                            return;
+                        }
+
+                        if (isCanceled) {
```

Comment:
do we really need isCanceled? In every case when the dialog is dismissed, we should call dismiss listener. I could be missing something here.